CHANGELOG.md

```diff
@@ -640,7 +640,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 ### Features
 
-* support env write operations for sites opted in to beta env var experience ([#4751](https://github.com/netlify/cli/issues/4751)) ([e0695f9](https://github.com/netlify/cli/commit/e0695f94f0a54e6d31c627ea2b0b139895a21bc1))
+* support env write operations for sites opted into beta env var experience ([#4751](https://github.com/netlify/cli/issues/4751)) ([e0695f9](https://github.com/netlify/cli/commit/e0695f94f0a54e6d31c627ea2b0b139895a21bc1))
```

Comment:
I'm still working on `in to` / `into`. Happy to drop if these are objectionable.